src/main/java/org/kie/processmigration/rest/KieServerHealth.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
```

Comment:
Previous version we had `HealthStatus` which has been renamed to `KieServerHealthCheck` but other than that, we didn't have any class named like this, and as per GH, this is showing the class is new.
```suggestion
 * Copyright 2021 Red Hat, Inc. and/or its affiliates.
```